src/popup/src/App.vue

```diff
@@ -17,6 +17,22 @@ import { defineComponent } from 'vue';
 import HelloWorld from './components/HelloWorld.vue';
 export default defineComponent({
   components: { HelloWorld },
+  data() {
+    return {
+      showSnyk: true,
+    };
+  },
+  mounted() {
+    chrome.storage.local.get('showSnyk').then(({ showSnyk }) => (this.showSnyk = showSnyk));
```

Comment:
Using `chrome.storage.local`. Most of the docs use `chrome.storage.sync` so the settings will be synced with the account logged in another computer.